.github/CODEOWNERS

```diff
@@ -1,8 +1,99 @@
-/.github/ @hashgraph/release-engineering
-**/pom.xml @hashgraph/release-engineering
-/gradle/ @hashgraph/release-engineering
-gradlew @hashgraph/release-engineering
-gradlew.bat @hashgraph/release-engineering
-**/buildSrc/ @hashgraph/release-engineering
-**/gradle.* @hashgraph/release-engineering
-**/*.gradle.* @hashgraph/release-engineering
+###################################
+##### Global Protection Rule ######
+###################################
+# NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below.
+*                                               @hashgraph/release-engineering @hashgraph/release-engineering-managers
+
+#########################
+##### Hedera Node  ######
+#########################
+
+# Hedera Node Root Protections
+/hedera-node/                                   @hashgraph/hedera-services
+/hedera-node/README.md                          @hashgraph/hedera-services @hashgraph/release-engineering @hashgraph/release-engineering-managers
+
+# Hedera Node Deployments - Configuration & Grafana Dashboards
+/hedera-node/configuration/**                   @hashgraph/release-engineering @hashgraph/devops @hashgraph/hedera-services @hashgraph/platform-hashgraph
+/hedera-node/infrastructure/**                  @hashgraph/release-engineering @hashgraph/devops @hashgraph/hedera-services
+
+# Hedera Node Docker Definitions
+/hedera-node/docker/                            @hashgraph/hedera-services @hashgraph/release-engineering @hashgraph/release-engineering-managers
+
+# Hedera Node Modules
+/hedera-node/hedera-mono-service/               @hashgraph/hedera-services
+/hedera-node/hapi*/                             @hashgraph/hedera-services
+/hedera-node/hedera-admin*/                     @hashgraph/hedera-services @kimbor @povolev15
+/hedera-node/hedera-app*/                       @hashgraph/hedera-services @hashgraph/platform-base
```

Comment:
Why platform-base here?

Reply:
@poulok This is due to a request from Nick to have @hendrikebbers and @netopyr approve all changes to this code.